packages/graphql-tool-utilities/src/tests/generate-possible-types.test.js

```diff
@@ -0,0 +1,10 @@
+import generatePossibleTypes from '../generate-possible-types.js';
+import * as possibleTypes from './fixtures.js'
+import * as schema from './schema.json'
+
+describe('generatePossibleTypes', () => {
+    it('generates possible types', () => {
+        expect(generatePossibleTypes(schema)).toStrictEqual(expect.objectContaining(possibleTypes.possibleTypes));
```

Comment:
I know this is weird... any suggestions on how to make this code cleaner? ie. `possibleTypes.possibleTypes